### PR TITLE
Updating issues docs to integrate old commit.

### PR DESCRIPTION
**Pull request by simeoncran**
Sohum created some changes to these issues docs last year. I'm
using them as inspiration to update our docs, and to deprecate another
issue.

### CODE REVIEW

File: source/Issues/LP0008.md [hunk 1]
```diff
@@ -3,10 +3,12 @@
...
 # Lottie-Windows Warning LP0008
 
-The Lottie file contains layer effects. Layer effects are currently not supported.
+The Lottie file contains Layer Effects. Layer Effects are currently not supported
+by Lottie-Windows.
 
 ## Remarks
-Lottie-Windows does not currently support layer effects. It may be possible to work around this by modifying your After Effects project to avoid the use of layer effects.
+Lottie does not support Layer Effects on most platforms. It may be possible to
+work around this by modifying your After Effects project to avoid the use of layer effects.
```
Comment (eliezerpMS) [inline source/Issues/LP0008.md:11]: >layer effects [](start = 77, length = 13)

For consistency should this also be capitalized to "Layer Effects"

File: source/Issues/LT0001.md [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
 # Lottie-Windows Warning LT0001
 
-A rectangle has an animated size with a trim path that is not currently supported.
+A rectangle shape cannot be rendered if it has a trim path and its size is animated.
 
 ## Remarks
 The combination of an animated size with certain types of trim path is not yet supported 
 by Lottie-Windows. You may be able to work around it by adjusting your After Effects 
-file.
+file, for example, consider converting the rectangle to a path instead.
```
Comment (derobinsMSFT) [inline source/Issues/LT0001.md:11]: >, [](start = 4, length = 1)

Nit: semicolon.

Comment (simeoncran) [reply source/Issues/LT0001.md:11]: I'm not getting the rules you're using for semicolons. These are not independent clauses so I don't think they should be semicolonized, but I'm guessing you have a specific rule in mind. What is the rule?

---
In reply to: [427572074](https://github.com/windows-toolkit/Lottie-Windows/pull/260#discussion_r427572074) [](ancestors = 427572074)

Comment (derobinsMSFT) [reply source/Issues/LT0001.md:11]: They strike me as independent clauses, which is probably why I'm leaning toward using a semicolon here.

---
In reply to: [427592796](https://github.com/windows-toolkit/Lottie-Windows/pull/260#discussion_r427592796) [](ancestors = 427592796,427572074)

Comment (simeoncran) [reply source/Issues/LT0001.md:11]: I think the "for example" makes them not independent. AFAIK, they're independent if they could be separate sentences. Not saying I'm right though, so educate me if I'm mistaken.

---
In reply to: [427626401](https://github.com/windows-toolkit/Lottie-Windows/pull/260#discussion_r427626401) [](ancestors = 427626401,427592796,427572074)

**Pull request merged**